### PR TITLE
JFG-1055: remove old metric names support

### PR DESCRIPTION
**Pull request by anton-antonenko**

### CODE REVIEW

File: chassis/core/src/main/java/com/griddynamics/jagger/engine/e1/reporting/SummaryReporter.java [hunk 1]
```diff
@@ -277,39 +275,9 @@ private void fillTestSummaryAndScalabilityData(String sessionId) {
...
-                    workloadTaskData.setStdDevLatency(new BigDecimal(metricsForThisTest.get(metricEntity).getValue()));
-                    dataForScalabilityPlots.get(testEntity).put(StandardMetricsNamesUtil.LATENCY_STD_DEV_ID, metricsForThisTest.get(metricEntity).getValue());
-                }
-            }
-
             String testStatusComment = "";
             Decision testStatus = Decision.OK;
 
-            // Success rate
-            Decision testSuccessRateStatus = decisionMaker.decideOnTest(workloadTaskData);
```
Comment (dlatnikov) [inline chassis/core/src/main/java/com/griddynamics/jagger/engine/e1/reporting/SummaryReporter.java]: decisionMaker is not needed at all in this case

Comment (dlatnikov) [inline chassis/core/src/main/java/com/griddynamics/jagger/engine/e1/reporting/SummaryReporter.java]: as a result com.griddynamics.jagger.engine.e1.reporting.DefaultSessionStatusDecisionMaker and com.griddynamics.jagger.engine.e1.reporting.SessionStatusDecisionMaker are not needed any more

File: chassis/util/src/main/java/com/griddynamics/jagger/util/StandardMetricsNamesUtil.java [hunk 2]
```diff
@@ -177,21 +136,18 @@ public static Double parseLatencyPercentileKey(String metricName) {
...
-        synonyms.put(DURATION_OLD_ID, Arrays.asList(DURATION_ID, "Duration"));
-        synonyms.put(ITERATION_SAMPLES_OLD_ID, Arrays.asList(ITERATION_SAMPLES_ID, "Iterations"));
-
-        // new to old and very old :-)
-        synonyms.put(THROUGHPUT_ID, Arrays.asList(THROUGHPUT_OLD_ID, THROUGHPUT));
-        synonyms.put(LATENCY_ID, Arrays.asList(LATENCY_OLD_ID, LATENCY));
-        synonyms.put(LATENCY_STD_DEV_ID, Arrays.asList(LATENCY_STD_DEV_OLD_ID));
-        synonyms.put(DURATION_ID, Arrays.asList(DURATION_OLD_ID, "Duration"));
-        synonyms.put(ITERATION_SAMPLES_ID, Arrays.asList(ITERATION_SAMPLES_OLD_ID, "Iterations"));
+    private static List<String> getSynonyms(String metricName) {
```
Comment (dlatnikov) [inline chassis/util/src/main/java/com/griddynamics/jagger/util/StandardMetricsNamesUtil.java]: please, review if we need this method. looks like it is not required any more

File: chassis/core/src/main/java/com/griddynamics/jagger/engine/e1/reporting/SummaryReporter.java (hunk 1 above)
Comment (dlatnikov) [inline chassis/core/src/main/java/com/griddynamics/jagger/engine/e1/reporting/SummaryReporter.java]: https://issues.griddynamics.net/browse/JFG-1001

Comment (anton-antonenko) [inline chassis/core/src/main/java/com/griddynamics/jagger/engine/e1/reporting/SummaryReporter.java]: should I do it in this PR or create another one?

Comment (anton-antonenko) [inline chassis/core/src/main/java/com/griddynamics/jagger/engine/e1/reporting/SummaryReporter.java]: done

File: chassis/configuration/configuration/reporter/reporting.conf.xml [hunk 3]
```diff
@@ -143,14 +137,6 @@
                         <property name="template" value="/reporting/diagnostics-report.jrxml"/>
                     </bean>
                 </entry>
-                <entry key="workloadSessionComparison">
-                    <bean class="com.griddynamics.jagger.engine.e1.reporting.WorkloadSessionComparisonReporter"
-                          parent="abstractReportProviderBean">
-                        <property name="template"
-                                  value="/reporting/workload-session-comparison-report.jrxml"/>
```
Comment (dlatnikov) [inline chassis/configuration/configuration/reporter/reporting.conf.xml]: what about this file?

Comment (anton-antonenko) [inline chassis/configuration/configuration/reporter/reporting.conf.xml]: done

**Pull request merged**